public/angular/templates/incidents/show.html

```diff
@@ -23,41 +23,43 @@
           </div>
         </div>
         <div class="card-body">
-          <table class="table table-bordered">
+          <table class="table table-bordered incidentShow__table">
```

Comment:
is this a naming convention to have __ (double underscores)?

Reply:
Yeah its BEM